### ISSUE TITLE
About STL10 top-1 accuracy

### ISSUE THREAD

**Issue by GA-17a** [sthalles/PyTorch-BYOL#6]
Hi, thanks for your share. But I have a question about your STL10 top-1 accuracy. Is the accuracy '70.1' fine-tuning from your pre-trained model 'resnet-18_40-epochs' or training from scratch for 40 epochs?

**Comment by hafnerfe** [comment]
@GA-17a if you look in eval/eval.ipynb, it achieves a testing accuracy of 70.1%. 
However, the model .../resnet-18-80-epochs/.../model.pth is loaded. 
When I use the exact same script to pretrain the model using 40 epochs (i.e. training from scratch), I only achieve an accuracy of 66.5%. 
Seeing the other open issue, other people seem to have the same problem. Maybe there's an error in the documentation.

**Comment by GA-17a** [comment]
@hafnerfe Thanks for your reply.

**Comment by GA-17a** [comment]
@hafnerfe Thanks for your reply, and it solved my question.
